ui/src/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const maxRenderedOutputSizeBytes = 1024 * 1024;
```

Comment:
Can you add a comment about the purpose of this constant? That it's really just to prevent `JSON.parse` from crashing the browser